### PR TITLE
Changes for compatibility with spack

### PR DESCRIPTION
**Pull request by mmesiti**
Mainly:
- Attempted change in `sombrero.sh` to make it callable from wherever,
  reverted after discovering Mosè Giordano's spack package,
  which would be broken by that.
- Commented out whole content of MkFlags, to allow spack to take over.
- CC -> MPICC where relevant
- Readme.

Reminder: the thin branch is used as a base by shoplifter 
to create the full Sombrero application.

The product of shoplifter are then committed to the `shoplifted` branch.
There is no interference between the changes in this pull request
and shoplifter's actions,
so these changes just appear in the shoplifted branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,36 +8,54 @@ A benchmarking utility for high performance computing based on lattice field the
 
 ## Requirements
   
-  Requires an MPI library and compiler.
+Requires an MPI library and compiler.
 
 ## Set Up
```
Comment (edbennett) [inline README.md]: The addition of the "Setup with Spack" heading below has highlighted to me that every heading apart from this one (and "Miscellaneous") is a noun, but this is a verb. For consistency, could you update this to "Setup", please? Everything else looks great; I'll merge once that's done.

**Review by edbennett — CHANGES_REQUESTED** [review]
One very minor change requested in a comment.

File: README.md (hunk 1 above)
Comment (edbennett) [reply README.md]: Just noticed this comment is not where I placed it. I specifically mean changing line 13 here. (I.e. what you wrote was correct, it's the existing text that needed updating.)

Comment (mmesiti) [reply README.md]: Thanks for the clarification.
The next commit fixes that.

**Comment by mmesiti** [comment]
The last commit adds a missing verb to a sentence
.

**Review by edbennett — CHANGES_REQUESTED** [review]
Sorry for not noticing this in the first review: please could you capitalise MPI, Spack, and SOMBRERO?

**Review by edbennett — APPROVED** [review]

**Pull request merged**